nodejs/src/eventhub.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  earliestEventPosition,
+  EventHubConsumerClient,
+  EventHubProducerClient,
+  SubscriptionEventHandlers
+} from "@azure/event-hubs";
+import { BlobCheckpointStore } from "@azure/eventhubs-checkpointstore-blob";
+import {
+  ContainerClient,
+  StorageSharedKeyCredential
+} from "@azure/storage-blob";
+import {
+  context as otContext,
+  SpanStatusCode,
+  trace
+} from "@opentelemetry/api";
+import { getTracer } from "./tracing";
+import { getEnvironmentVariable } from "./utils";
+
+// TODO: work out the naming conventions of these spans.
+const eventHubHandlers: SubscriptionEventHandlers = {
+  processEvents: async (events, context) => {
+    if (events.length === 0) {
+      return;
+    }
+
+    return getTracer().startActiveSpan(
+      "eventhub-processEvents",
+      async (span) => {
+        for (const event of events) {
+          console.log(`Received event: ${JSON.stringify(event.body)}`);
+        }
+        await context.updateCheckpoint(events[events.length - 1]);
+        span.end();
+      }
+    );
+  },
+  processError: (err) => {
+    getTracer().startActiveSpan("eventhub-processError", (span) => {
```

Comment:
this should not be a span, we should not create any spans in this test app - the assumption is that spans created by SDKs are enough. Why are we crating any?

Reply:
From my perspective I figured as an application developer I would want to know where this ties into my code - for example let's say the eventhub producer span signals an error, I would ideally want to tie this to an entrypoint in my client code (if I use the eventhub producer in multiple places). 

But... I really am just guessing here, you're the expert 😄 so I can remove the app level spans if that's incorrect

Reply:
the error should be reported by SDK on the processing span (if possible). The idea is that SDK should provide right enough observability for generic sceanrios. 
The majority of users don't want to write any tracing code and SDK should help them